# Take EasyBuild compiler flags for C, C++, and Fortran
# OCT 5th, 2020 by B. Hajgato (UGent)
diff -ru DIRAC-19.0-Source.orig/cmake/custom/compiler_flags/Intel.C.cmake DIRAC-19.0-Source/cmake/custom/compiler_flags/Intel.C.cmake
--- DIRAC-19.0-Source.orig/cmake/custom/compiler_flags/Intel.C.cmake	2019-12-12 17:26:15.000000000 +0100
+++ DIRAC-19.0-Source/cmake/custom/compiler_flags/Intel.C.cmake	2020-10-07 19:53:57.517988580 +0200
@@ -1,5 +1,5 @@
 if(CMAKE_C_COMPILER_ID MATCHES Intel)
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -wd981 -wd279 -wd383 -wd1572 -wd177")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -wd981 -wd279 -wd383 -wd1572 -wd177")
     set(CMAKE_C_FLAGS_RELEASE "-O2")
     set(CMAKE_C_FLAGS_DEBUG "-O0")
 endif()
diff -ru DIRAC-19.0-Source.orig/cmake/custom/compiler_flags/Intel.CXX.cmake DIRAC-19.0-Source/cmake/custom/compiler_flags/Intel.CXX.cmake
--- DIRAC-19.0-Source.orig/cmake/custom/compiler_flags/Intel.CXX.cmake	2019-12-12 17:26:15.000000000 +0100
+++ DIRAC-19.0-Source/cmake/custom/compiler_flags/Intel.CXX.cmake	2020-10-07 19:54:10.338306645 +0200
@@ -1,5 +1,5 @@
 if(CMAKE_CXX_COMPILER_ID MATCHES Intel)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unknown-pragmas")
-    set(CMAKE_CXX_FLAGS_RELEASE "-debug -O3 -DNDEBUG")
+    set(CMAKE_CXX_FLAGS_RELEASE "-O2 -DNDEBUG")
     set(CMAKE_CXX_FLAGS_DEBUG "-O0 -debug -DDEBUG")
 endif()
diff -ru DIRAC-19.0-Source.orig/cmake/custom/compiler_flags/Intel.Fortran.cmake DIRAC-19.0-Source/cmake/custom/compiler_flags/Intel.Fortran.cmake
--- DIRAC-19.0-Source.orig/cmake/custom/compiler_flags/Intel.Fortran.cmake	2019-12-12 17:26:15.000000000 +0100
+++ DIRAC-19.0-Source/cmake/custom/compiler_flags/Intel.Fortran.cmake	2020-10-07 19:55:34.670398011 +0200
@@ -1,5 +1,5 @@
 if(CMAKE_Fortran_COMPILER_ID MATCHES Intel)
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -w -assume byterecl -g -traceback -DVAR_IFORT")
-    set(CMAKE_Fortran_FLAGS_RELEASE "-O3 -ip")
+    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -w -assume byterecl -DVAR_IFORT")
+    set(CMAKE_Fortran_FLAGS_RELEASE " ")
     set(CMAKE_Fortran_FLAGS_DEBUG "-O0")
 endif()
